.github/workflows/release.yml

```diff
@@ -30,6 +30,12 @@ jobs:
   publish:
     name: Build package
     runs-on: macos-11
+    env:
+      PUB_KEY: ${{ secrets.PUB_KEY }}
+      SUB_KEY: ${{ secrets.SUB_KEY }}
+      PAM_PUB_KEY: ${{ secrets.PAM_PUB_KEY }}
+      PAM_SUB_KEY: ${{ secrets.PAM_SUB_KEY }}
+      PAM_SECRET_KEY: ${{ secrets.PAM_SECRET_KEY }}
```

Comment:
@mohitpubnub I believe this is not required, because those expected to be used in test cases only, not ion library core.